helper_dart/lib/src/insights.dart

```diff
@@ -17,44 +17,45 @@ class Insights implements EventTracker {
   static const _maxObjectIDsPerEvent = 20;
   static const _maxFiltersPerEvent = 10;
 
+  /// Set custom user token
+  static String get userToken => _userTokenController.userToken;
+
+  static set userToken(String userToken) {
+    _userTokenController.userToken = userToken;
+  }
+
+  /// Determines whether the value is stored in memory or persistent storage.
+  static set allowPersistentUserTokenStorage(bool isAllowed) {
+    _userTokenController.allowPersistentUserTokenStorage = isAllowed;
+  }
+
   /// Map storing Insights instances per application ID.
   static final Map<String, Insights> _insightsPool = <String, Insights>{};
 
   /// Entity managing the user token generation and storage
-  static final _userTokenController = UserTokenController();
+  static final _userTokenController = UserTokenStorage();
```

Comment:
shouldn't this be called _userTokenStorage now?

Reply:
Yes, it's a left-over.